### DISCUSSION TITLE
Owndroid Spanish translation

### DISCUSSION THREAD

**Discussion by nubesurrealista** [BinTianqi/OwnDroid#50] [General]
Greetings

I am currently translating Owndroid strings **(English)** into **Spanish**. This is longer than I expected so it will take a few days (because of other occupations) but, I will finish them. I have advanced in translating almost half of the strings. I think being translated into several languages will help in the visibility of the app, so I will do my part  n.n

**Comment by nubesurrealista** [comment]
Btw, I can translate the readme from this repository as well, if you consider it necessary. In this it would take much less

**Comment by BinTianqi** [comment]
Big thanks!

> Can you translate those strings to Spanish a few days later? I think some English strings are incorrect, I have to correct them

Update: I think I correct most of the English strings, you can start translating now
